addons/mail/models/discuss/discuss_channel.py

```diff
@@ -1085,9 +1090,7 @@ def _get_or_create_chat(self, partners_to, pin=True):
                     Command.create({
                         'partner_id': partner_id,
                         # only pin for the current user, so the chat does not show up for the correspondent until a message has been sent
-                        # manually set the last_interest_dt to make sure that it works well with the default last_interest_dt (datetime.now())
                         'unpin_dt': False if partner_id == self.env.user.partner_id.id else fields.Datetime.now(),
-                        'last_interest_dt': fields.Datetime.now() if partner_id == self.env.user.partner_id.id else fields.Datetime.now() - timedelta(seconds=30),
```

Comment:
Note: `last_interest_dt` being a bit in the past is moved to field default to avoid having to worry about it in code/tests every time we create one (1 sec in the past makes no functional difference and fixes this annoying issue).